src/main/groovy/edu/wpi/first/gradlerio/wpi/dependencies/WPIDepsExtension.groovy

```diff
@@ -0,0 +1,82 @@
+package edu.wpi.first.gradlerio.wpi.dependencies
+
+import edu.wpi.first.gradlerio.wpi.WPIExtension
+import groovy.transform.CompileStatic
+
+@CompileStatic
+public class WPIDepsExtension {
+
+    final WPIExtension wpi
+
+    WPIVendorDepsExtension vendor
+
+    WPIDepsExtension(WPIExtension wpi) {
+        this.wpi = wpi
+        this.vendor = new WPIVendorDepsExtension(this)
+    }
+
+    List<String> wpilib() {
+        wpilibJni().each {
+            wpi.project.dependencies.add("nativeZip", it)
+        }
+        wpilibDesktopJni().each {
+            wpi.project.dependencies.add("nativeDesktopZip", it)
+        }
+        return wpilibJars()
+    }
+
+    List<String> wpilibJars() {
+        return ["edu.wpi.first.wpilibj:wpilibj-java:${wpi.wpilibVersion}".toString(),
+                "edu.wpi.first.ntcore:ntcore-java:${wpi.wpilibVersion}".toString(),
+                "edu.wpi.first.wpiutil:wpiutil-java:${wpi.wpilibVersion}".toString(),
+                "edu.wpi.first.thirdparty.frc2019.opencv:opencv-java:${wpi.opencvVersion}".toString(),
+                "edu.wpi.first.cscore:cscore-java:${wpi.wpilibVersion}".toString(),
+                "edu.wpi.first.cameraserver:cameraserver-java:${wpi.wpilibVersion}".toString(),
+                "edu.wpi.first.hal:hal-java:${wpi.wpilibVersion}".toString()]
```

Comment:
You could simplify a lot of this with a `[].map { it.toString() }`

Reply:
It's not worth the extra iteration IMO